pkgs/filestash/default.nix

```diff
@@ -1,4 +1,18 @@
-{ dream2nix, stdenv, self, python, runCommand, pkgs, filestash-src, glib, vips, libraw, pkg-config, buildGoModule }:
+{ buildGo120Module
```

Comment:
22.11's `buildGoModule` points to `buildGo119Module` and Filestash theoretically requires go 1.20 in https://github.com/mickael-kerjean/filestash/blob/26ee2006f4b0b5af5335e92238fd4a24c1a832c0/go.mod#L3

Filestash does however build just fine with `buildGo119Module` rn.
There just isn't any go 1.20 specific code in there yet.
This would more of an issue if Filestash wouldn't vendor the go dependencies afaik.

So yeah, dunno.
unstable and the upcoming 23.05 both use go 1.20 by default.
Not sure if switching back to unstable is worth it, though.
But that input isn't getting auto-updated anyway.

tl;dr: it doesn't matter right now, but it could in the future
 
Opinions?

---

lol I am getting http/500s trying to comment this :eyes:

Guess I have some time asking something else:
Do you think Filestash has a chance to get upstreamed into nixpkgs if Filestash starts providing a `package-lock.json`?

Update: https://www.githubstatus.com/incidents/52z0j6phhnjs has been resolved :)

Reply:
With that in mind, do you want me to change it back to `buildGoModule` or keep `buildGo120Module`? :)

Reply:
Yeah, in Nixpkgs for example they would want to use the latest buildGoModule, we should only pin the version if there is a legitimate observable compilation or runtime issue, same for all of the other libraries(inputs) involved